tsl/src/bgw_policy/job.c

```diff
@@ -770,7 +785,11 @@ job_execute(BgwJob *job)
 	if (!ActiveSnapshotSet())
 	{
 		pushed_snapshot = true;
+#if (PG_VERSION_NUM >= 120800) || (PG_VERSION_NUM >= 130400)
```

Comment:
```suggestion
#if (PG12 && PG_VERSION_NUM >= 120800) || (PG13 && PG_VERSION_NUM >= 130400) || PG14_GE
```
Have to include compat/compath.h for PGXX macros to work

Reply:
Fixed, thanks!